packages/store/src/usersSlice/thunks.ts

```diff
@@ -669,7 +785,7 @@ export const saveGlobalSettings = createAsyncThunk(
         return GeneralApi.post(`${useradmApiUrl}/settings`, updatedSettings, { headers })
           .then(() => {
             if (notify) {
-              tasks.push(dispatch(setSnackbar('Settings saved successfully')));
+              dispatch(setSnackbar('Settings saved successfully'));
             }
```

Comment:
why?

Reply:
It was done to avoid mixing different types in `tasks` array, but I will fix it